patches/chrome-android-BUILD.gn.patch

```diff
@@ -50,11 +50,35 @@ index 616c88364d599be02afacc8d0d73f00bad374a82..2c6d081ef5be5adf6fd1b999c9832372
        if (_is_monochrome) {
          deps += [ "//android_webview:locale_pak_assets" ]
          if (webview_includes_weblayer && !_is_bundle_module) {
-@@ -2647,6 +2653,7 @@ android_library("base_module_java") {
+@@ -2454,6 +2460,7 @@ static_library("browser_test_support") {
+     "//url:gurl_android_test_helper",
+   ]
+ }
++_lib_dir = get_label_info("//brave/third_party/android_deps:com_wireguard_android($default_toolchain)","target_out_dir") + "/com_wireguard_android_java/jni"
+ 
+ # Defines a target that derives from the chrome public application. This
+ # can be either an APK or an app bundle module. This supports
+@@ -2506,6 +2513,7 @@ template("chrome_public_apk_or_module_tmpl") {
+     load_library_from_apk = chromium_linker_supported
+ 
+     version_name = chrome_version_name
++    deps += ["//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar"]loadable_modules = [ "$_lib_dir/$android_app_abi/libwg-go.so"]
```

Comment:
same as below, don't hardcode the actuals deps and please leave a space between the end of deps list and loadable_modules